src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationService.kt

```diff
@@ -134,4 +142,25 @@ class CalculationService(
       throw PreconditionFailedException("The booking data used for the preliminary calculation has changed")
     }
   }
+
+  @Transactional(readOnly = true)
+  @Suppress("TooGenericExceptionCaught")
+  fun writeToNomis(bookingId: Long, calculation: BookingCalculation) {
+    val calculationRequest = calculationRequestRepository.findById(calculation.calculationRequestId)
+      .orElseThrow { EntityNotFoundException("No calculation request exists") }
+    val updateOffenderDates = UpdateOffenderDates(
+      calculationUuid = calculationRequest.calculationReference,
+      submissionUser = "YMUSTAFA_GEN",
+      keyDates = OffenderKeyDates(calculation.dates[CRD], calculation.dates[LED], calculation.dates[SED])
```

Comment:
I know - its a temp PR! But you'll probably need to get this from the received token.